packages/neon_framework/lib/src/storage/request_cache.dart

```diff
@@ -65,15 +66,15 @@ final class DefaultRequestCache implements RequestCache {
     final cacheDir = await getApplicationCacheDirectory();
     database = await openDatabase(
       p.join(cacheDir.path, 'cache.db'),
-      version: 3,
+      version: 4,
       onCreate: onCreate,
       onUpgrade: (db, oldVersion, newVersion) async {
         // We can safely drop the table as it only contains cached data.
         // Non breaking migrations should not drop the cache. The next
         // breaking change should remove all non breaking migrations before it.
         await db.transaction((txn) async {
-          if (oldVersion <= 2) {
-            await txn.execute('DROP TABLE cache');
```

Comment:
The old migration needs to stay because coming from an older version should still migrate to the new schema.
Only if we absolutely need to drop the table we can remove all older migrations (we also discussed this last weekend).